syntax = "proto2";

package ships;

import "i18n.proto";

message Ships {
    message Ship {
        required i18n.I18N name = 1;
        required int32 groupID = 2;
        required bool published = 3;
        required int32 highSlotNum = 4;
        required int32 medSlotNum = 5;
        required int32 lowSlotNum = 6;
        required int32 rigSlotNum = 7;
        required bool hasSubsystem = 8;
        required int32 turretSlotNum = 9;
        required int32 launcherSlotNum = 10;
        required int32 droneBandwidth = 11;
    }

    map<int32, Ship> entries = 1;
}
